### ISSUE TITLE
Aspect Ratio: Android Beta and Stable - Don't think it ever worked

### ISSUE THREAD

**Issue by SyCoREAPER** [yuliskov/SmartTube#3454]
On Android (any version) aspect Ratio doesn't seem to work, or maybe I'm not understanding the intended use.

My tablet for example is 16:10, if I set SmartTube to 16:10 it stretches and distorts videos to fit, using the zoom to "fit to height" does a crop instead to fill but maintain the proper ratio of width to height.

So if I am not using it correctly, could you elaborate what it's for? If it is meant to be set to match the device, I assume it isn't working correctly.

**Comment by KneesDev** [comment]
[Device support](https://github.com/yuliskov/SmartTube?tab=readme-ov-file#device-support) & [Compatibility](https://github.com/yuliskov/SmartTube?tab=readme-ov-file#compatibility)
> Though this app technically runs on smartphones and tablets, it is not optimized for such and offers no official support!

The app isn't even meant to be used on tablets, only TVs. Read before using.

**Comment by SyCoREAPER** [comment]
> [Device support](https://github.com/yuliskov/SmartTube?tab=readme-ov-file#device-support) & [Compatibility](https://github.com/yuliskov/SmartTube?tab=readme-ov-file#compatibility)
> > Though this app technically runs on smartphones and tablets, it is not optimized for such and offers no official support!
> 
> The app isn't even meant to be used on tablets, only TVs. Read before using.

Doesn't mean you need to be a fat-hole because I missed that. Was a helpful response until the end.

**Comment by KneesDev** [comment]
You missed the whole repository description too?
![image](https://github.com/yuliskov/SmartTube/assets/74821251/c0301749-25ea-44fb-88d0-43f58776ccc5)
As well as the second line in the README?
![image](https://github.com/yuliskov/SmartTube/assets/74821251/b13cd4e8-4dd5-46e2-b0cf-d79772a7493e)

I could've said more, but I decided not to be so toxic towards you.

**Comment by SyCoREAPER** [comment]
> You missed the whole repository description too?
> ![image](https://github.com/yuliskov/SmartTube/assets/74821251/c0301749-25ea-44fb-88d0-43f58776ccc5)
> As well as the second line in the README?
> ![image](https://github.com/yuliskov/SmartTube/assets/74821251/b13cd4e8-4dd5-46e2-b0cf-d79772a7493e)
> 
> I could've said more, but I decided not to be so toxic towards you.

I'm well aware what it's intended for. I only recently started using it on mobile. I know it's not designed for it and has limited functionality but it's not always clear what is and isn't for what.

Not to mention, what idiot would have an Android TV-style box hooked up to a 16:10 screen?

Point is, just as you assumed I didn't read or know what I'm doing, I assumed based on your, while not toxic, passive aggressive comment was to stir things up.

It's a harmless question that could have been addressed and been done with instead of this back and forth with just the first part of your statement.

Lets just agree to part ways and be done with it. Question has been answered so I'm closing it and ignoring all further replies (from anyone).
